contracts/src/v0.8/llo-feeds/v0.4.0/test/verifier/VerifierVerifyTest.t.sol

```diff
@@ -0,0 +1,577 @@
+
+         V3Report memory reportT3 = V3Report({
+            feedId: FEED_ID_V3,
+            observationsTimestamp: OBSERVATIONS_TIMESTAMP,
+            validFromTimestamp: uint32(block.timestamp),
+            nativeFee: uint192(DEFAULT_REPORT_NATIVE_FEE),
+            linkFee: uint192(DEFAULT_REPORT_LINK_FEE),
+            expiresAt: uint32(block.timestamp),
+            benchmarkPrice: MEDIAN,
+            bid: BID,
+            ask: ASK
+        });
+
+         // testing we can verify reports with ConfigC signers
+         signedReport = _generateV3EncodedBlob(reportT3, s_reportContext, reportSignersConfigC);
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+
+         //  testing an old report (block timestamp) with ConfigC signers should  verify via ConfigB
+         signedReport = _generateV3EncodedBlob(reportT2, s_reportContext, reportSignersConfigC);
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         // deactivating to make sure we are really verifiying via ConfigB
+         s_verifier.setConfigActive(1, false);
+         vm.expectRevert(abi.encodeWithSelector(DestinationVerifier.BadVerification.selector));
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         s_verifier.setConfigActive(1, true);
+
+         // testing a recent report with ConfigA signers should not verify
+         signedReport = _generateV3EncodedBlob(reportT3, s_reportContext, reportSignersConfigA);
+         vm.expectRevert(abi.encodeWithSelector(DestinationVerifier.BadVerification.selector));
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+
+         // testing an old report (block timestamp) with ConfigA signers should  verify via ConfigB
+         signedReport = _generateV3EncodedBlob(reportT2, s_reportContext, reportSignersConfigA);
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         // deactivating to make sure we are really verifiying via ConfigB
+         s_verifier.setConfigActive(1, false);
+         vm.expectRevert(abi.encodeWithSelector(DestinationVerifier.BadVerification.selector));
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         s_verifier.setConfigActive(1, true);
+
+         // testing an old report (block timestamp) with ConfigA signers should  verify via ConfigA
+         signedReport = _generateV3EncodedBlob(reportT1, s_reportContext, reportSignersConfigA);
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         // deactivating to make sure we are really verifiying via ConfigB
+         s_verifier.setConfigActive(0, false);
+         vm.expectRevert(abi.encodeWithSelector(DestinationVerifier.BadVerification.selector));
+         s_verifierProxy.verify(signedReport, abi.encode(native));
+         s_verifier.setConfigActive(0, true);
+
+    }
+
+
+    function test_verifyFailsWhenReportIsOlderThanConfig() public {
```

Comment:
@Fletch153  this test currently fails. I dont think it is a biggie, but worth checking if it is a case we should guard against